.github/workflows/ci.yml

```diff
@@ -24,10 +24,15 @@ jobs:
           java-version: ${{ matrix.jvm-version }}
       - name: Assemble main and test classes
         run: ./gradlew classes testClasses
-      - name: Test
-        run: ./gradlew test
       - name: Check
-        run: ./gradlew check
+        shell: pwsh
```

Comment:
Why do you need the powershell?

Reply:
Because in a previous commit I've executed the checks written in "run" using a unix shell syntax and they didn't work on windows, so I've changed the implementation.
Probably using the docker gradle plugin you've suggested on the other comment this could be removed